tests/periph_spi/main.c

```diff
@@ -488,13 +480,31 @@ int cmd_bench(int argc, char **argv)
     }
     stop = xtimer_now_usec();
     sched_stop = _sched_ticks();
-    sched_diff_us = xtimer_usec_from_ticks(xtimer_diff(sched_stop,
-                                                       sched_start));
+    sched_diff_us = _xtimer_diff_usec(sched_stop, sched_start);
     printf("14 - transfer %i times %i byte to register:", BENCH_REDOS, BENCH_LARGE);
     printf("%"PRIu32"\t%"PRIu32"\n", (stop - start), sched_diff_us);
     sum += (stop - start);
     sched_sum += sched_diff_us;
 
+    /* 15 - acquire & release the bus 1000 times */
```

Comment:
technically this releases and acquires the bus 1000 times :smile: 
```suggestion
    /* 15 - release & acquire the bus 1000 times */
```